tools/bouncer_worker/src/lib/runCommand.js

```diff
@@ -4,40 +4,34 @@ const { ERRCODE_TIMEOUT, ERRCODE_UNKNOWN_ERROR } = require('../constants/errorCo
 const logger = require('./logger');
 const processMonitor = require('./processMonitor');
 const { timeoutMS } = require('./config').config;
+const processReporting = require('./config').config.processMonitoring.enabled;
 
 const run = (
 	exe,
 	params,
 	{ codesAsSuccess = [], verbose = true, logLabel },
-	memoryReporting = { enabled: false, maxMemory: 0, processTime: 0 },
+	processInformation,
 ) => new Promise((resolve, reject) => {
-	if (verbose) logger.info(`Executing command: ${exe} ${params.join(' ')} memoryReporting: ${memoryReporting.enabled}`, logLabel);
+	if (verbose) logger.info(`Executing command: ${exe} ${params.join(' ')} processMonitoring: ${processReporting}`, logLabel);
 	const cmdExec = spawn(exe, params, { shell: true });
-	if (memoryReporting.enabled) {
-		processMonitor.startMonitor(cmdExec.pid);
+	if (processReporting) {
```

Comment:
```suggestion
	if (processReporting && !processInformation.doNotMonitor) {
```

Reply:
instead of having a special flag for do not monitor, isn't it better to just pass undefined for processInformation to denote that?